test/07_ssl/setup.py

```diff
@@ -1,17 +1,9 @@
 import ssl
-import sys
+import sys, urllib.request
 from setuptools import setup, Extension
 
-if sys.version_info[0] == 2:
```

Comment:
This file need to be python 2.7 compatibile. We still support build python 2.7 wheels.

Reply:
Huh.
Something went wrong there. I had already undone this change...

Reply:
Too many rebases, I guess.

Reply:
OK, I know what went wrong. I rebased and merged an old version of the commit, after I had fixed it, by merging your PR :-D

My bad. Too much rebasing!